guides/get_action_context.md

```diff
@@ -0,0 +1,50 @@
+# Get action context information 
+
+The App needs access to the ActionContext via ActionToken to provide corresponding configuration page with the specific action context.
+
+### Path
+```
+GET /v1/apps/{appId}/getActionContext/{actionHandlerContextToken}
+```
+actionHandlerContextToken has only 5 minutes liftime, and can only be used once. It will be deleted once it is used. 
+
+#### Parameters
+
+|Type|Name|Description|Schema|
+|---|---|---|---|
+|**Header**|Authorization required|Authorization header, in form of Bearer {access_token}. The access_token is issued as result of an oauth flow by Watson Work Services|String|
+|**Path**|appId required|Unique id of an app|String|
+|**Path**|actionHandlerContextToken required|token generated by caller before|String|
+
+#### Responses
```

Comment:
this area is incorrect, we can't reference configuration, we need instead to delineate the different actions and what parameters, etc are returned

Reply:
This is just the rest api description for get action context, because action handler have the similiar process as original configuration api, so does this getActionContext, only the returned json object is different from configuration api, as it need to support more actions. We have listed the possible format of returned json object (https://github.com/watsonwork/watsonwork-developer-docs/pull/100/files/a411f50b36147bbcc0235a8bf4a5461959edc719#diff-ee8e307fae4d0de73e97254c586f230bR45) as we only support space config and team config for now.
You mentioned `different actions and parameters` are for setActionHandler api, when user trigger some action  from watson work space, it will first set action action with action type and parameters and return the action token back, then client will open the action url with the token as parameter, the app will call this getActionContext to retrieve the related action context of this action. So setActionContext(which need action type and parameters) is not the public api exposed to app, only this getActionContext is public for app to call.

Reply:
yes